Lottery-Game-Chainlink-VRF.md

```diff
@@ -318,7 +318,7 @@ This function first checks if our contract has Link token before we request for
     }
 ```
 
-This function was inherited from `VRFConsumerBase`. It is called by `VRFCoordinator` contract after it recieves the randomness from the external world. After recieving the randomness which can be any number in the range of uint256 we decrease its range from `0 to players.length-1` using the mod operaator
+This function was inherited from `VRFConsumerBase`. It is called by `VRFCoordinator` contract after it receives the randomness from the external world. After receiving the randomness which can be any number in the range of uint256 we decrease it's range from `0 to players.length-1` using the mod operator.
```

Comment:
This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).